cmd/postgres_exporter/main.go

```diff
@@ -146,9 +145,8 @@ func main() {
 
 	http.HandleFunc("/probe", handleProbe(logger))
 
-	level.Info(logger).Log("msg", "Listening on address", "address", *listenAddress)
```

Comment:
Does the exporter toolkit log the listen address? That seems useful for users. Cuts down on issues filed.

Reply:
Yup, the new version of the toolkit handles the logging, since the listen address can be called multiple times.